### ISSUE TITLE
Basic Example

### ISSUE THREAD

**Issue by theglowingman** [rikas/zoho_hub#4]
Hello, could you please provide a basic example of setting up and connection with an Auth token and then requesting the CurrentUser from zoho?  I'm not that great of a coder and I'm struggling to understand how the wrapper works.  Thanks.

**Comment by TimCummings** [comment]
I've only just dipped my toes into this project so far, but I'll see if I can put something together to add to the README.

Until then, I'll add that most of the progress I've made getting authenticated and connected came from reading Zoho's API documentation and isolating steps of the process to do sanity checks with a HTTP utility like Postman.

**Comment by rikas** [comment]
@TimCummings any progress on that? It would be awesome!

**Comment by TimCummings** [comment]
Sorry, it's taken longer than I expected to got to this. I've got a working record class now, so it's just an issue of compiling my notes and filling in any blanks pertaining to the requested example. I went ahead and updated the README with general info and things that were particularly helpful to me. Most of the major pieces to get started should be there, but I'll try to distill a simple and concise example as requested over the next few days.
